#Button LED (Multicolor)
uint8 NO_STATUS_CHANGE = 0
uint8 OFF = 1
uint8 ON = 2   #ONLY FOR FIRE LED
uint8 GREEN = 2
uint8 YELLOW = 3
uint8 RED = 4

#Button_IDs
uint8 LED_A = 0
uint8 LED_B = 1
uint8 LED_D = 2
uint8 LED_E = 3
uint8 LED_T12 = 4
uint8 LED_T34 = 5
uint8 LED_T56 = 6
uint8 LED_POV2 = 7
uint8 LED_I = 8


uint8 led_fire  #Supports only ON and OFF!
uint8[9] color_leds
